File: Capgemini/Mohit Shikre/Assignment4/product/src/app/product.ts
export class Product {
  id: number;
  title: string;
  description: string;
  sale_price: number;
  current_stock: number;
  constructor() {
    this.id = 0;
    this.title = "";
    this.description = "";
    this.sale_price = 0;
    this.current_stock = 0;
  }
}

File: Capgemini/Team-8/Requirement_1/Requirement_1/Models/Product.cs
using System.ComponentModel.DataAnnotations;

namespace Requirement_1.Models
{
    public class Product
    {
        public int Id { get; set; }

        [Required(ErrorMessage = "Product title is required !")]
        [StringLength(64, ErrorMessage = "Maximum of 64 Characters only !")]
        [MinLength(2, ErrorMessage = "Minimum of 2 Characters required !")]

        public string Product_title { get; set; }

        [Required(ErrorMessage = "description is required !")]
        [StringLength(64, ErrorMessage = "Maximum of 64 Characters only !")]
        [MinLength(2, ErrorMessage = "Minimum of 2 Characters required !")]

        public string Description { get; set; }

        public int Sale_Price { get; set; }
    }
}
